libraries/AP_NavEKF/AP_NavEKF_core_common.h

```diff
@@ -49,4 +49,19 @@ class NavEKF_core_common {
 
     // fill all the common scratch variables with NaN on SITL
     void fill_scratch_variables(void);
+
+    // zero part of an array
```

Comment:
maybe 
`// zero part of an array boundaries included [n1, n2]`
to clear ambiguity when seeing the function definition on IDE ?

Reply:
fixed